src/pages/quote/quote.css

```diff
@@ -0,0 +1,3 @@
+.quote {
+  color: black;
+}

```

Comment:
- [ ] **Optional:** For fewer lines like this, you could use [inline styling](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/react_basic_styling.md#inline-styles) rather than having a file for them.